### PR TITLE
Feature/jansluk/98 link toggl and plan projects

### PR DESCRIPTION
**Pull request by DuplosFidibuss**

### CODE REVIEW

File: AcademicTimePlanner/Data/DisplayData.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        public List<PlanProject> PlanProjects { get; }
+
+        public double TotalTrackedTime { get; }
+
+        public double TotalPlannedTime { get; }
+
+        public double RemainingDuration { get; }
+
+        /// <summary>
+        /// This class holds all data needed for the graphical overview of planning and tracked time data.
```
Comment (Ra1nerZufall) [inline AcademicTimePlanner/Data/DisplayData.cs:22]: We have to  change all the descriptions.
They will be all out of date by now.

File: AcademicTimePlanner/DataMapping/Plan/PlanTask.cs [hunk 2]
```diff
@@ -10,27 +10,14 @@ public class PlanTask
 
         public string? Name { get; set; }
 
-        public long TogglId { get; set; }
+        public List<long> TogglIds { get; set; }
```
Comment (Ra1nerZufall) [inline AcademicTimePlanner/DataMapping/Plan/PlanTask.cs:13]: I will probably try to change this to a dictionary so it matches the rest.

**Review by Ra1nerZufall — DISMISSED** [review]

File: AcademicTimePlanner/DataMapping/Plan/PlanTask.cs (hunk 2 above)
Comment (Ra1nerZufall) [reply AcademicTimePlanner/DataMapping/Plan/PlanTask.cs]: I changed this please check if ok

Comment (Ra1nerZufall) [reply AcademicTimePlanner/DataMapping/Plan/PlanTask.cs]: sent you the updated testProject_3 JSON. The two others remain the same as they did not have any tasks to begin with.

File: AcademicTimePlanner/Data/DisplayData.cs (hunk 1 above)
Comment (DuplosFidibuss) [reply AcademicTimePlanner/Data/DisplayData.cs:22]: We are going to look at the code documentation anyway next week. We can bring that up again then.

File: AcademicTimePlanner/DataMapping/Plan/PlanTask.cs (hunk 2 above)
Comment (DuplosFidibuss) [reply AcademicTimePlanner/DataMapping/Plan/PlanTask.cs]: Looks good to me.

**Comment by DuplosFidibuss** [comment]
@Ra1nerZufall You have to approve once again because of the changes that you have made.

**Review by Ra1nerZufall — APPROVED** [review]

**Pull request merged**